packages/account/src/providers/transaction-summary/input.ts

```diff
@@ -33,16 +34,23 @@ export function getInputsContract(inputs: Input[]) {
 }
 
 /** @hidden */
-export function getInputFromAssetId(inputs: Input[], assetId: string) {
+export function getRelevantInputs(inputs: Input[], assetId: string = ZeroBytes32) {
   const coinInputs = getInputsCoin(inputs);
   const messageInputs = getInputsMessage(inputs);
-  const coinInput = coinInputs.find((i) => i.assetId === assetId);
-  // TODO: should include assetId in InputMessage as well. for now we're mocking ETH
-  const messageInput = messageInputs.find(
-    (_) => assetId === '0x0000000000000000000000000000000000000000000000000000000000000000'
-  );
+  let coinInput = coinInputs.find((i) => i.assetId === assetId);
+  // #TODO: There are times when the ReceiptCall's baseAssetId doesn't match the CoinInput's assetId
```

Comment:
In my experience testing this with the [connectors](https://connectors.fuel.network/), I've found that the [ReceiptCall](https://docs.fuel.network/docs/specs/abi/receipts/#call-receipt) 's `baseAssetId` was the zero 32-byte default. This doesn't seem to be accurate though given the predicate one may be interacting with would have a different `baseAssetId`. This method needs to be flexible enough to handle multiple assets within multiple `CoinInput` messages.